### PR TITLE
Add Discord invitation link

### PR DESCRIPTION
**Pull request by kokofixcomputers**
Add discord server

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # Sandboxie documentation
+[![Join our Discord Server](https://img.shields.io/badge/Join-Our%20Discord%20Server%20for%20bugs%20,%20feedback%20and%20more!-blue?style=for-the-badge&logo=discord)](https://discord.gg/TZDC3qbCSk)
```
Comment (isaak654) [inline README.md]:
```suggestion

[![Join our Discord Server](https://img.shields.io/badge/Join-Our%20Discord%20Server%20for%20bugs%20,%20feedback%20and%20more!-blue?style=for-the-badge&logo=discord)](https://discord.gg/TZDC3qbCSk)
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Review by isaak654 — APPROVED** [review]

**Comment by kokofixcomputers** [comment]
Thanks @isaak654

File: README.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
 # Sandboxie documentation
+
+[![Join our Discord Server](https://img.shields.io/badge/Join-Our%20Discord%20Server%20for%20bugs%20,%20feedback%20and%20more!-blue?style=for-the-badge&logo=discord)](https://discord.gg/TZDC3qbCSk)
```
Comment (isaak654) [inline README.md:3]:
```suggestion
[![Join our Discord Server](https://img.shields.io/badge/Join-Our%20Discord%20Server%20for%20bugs%20,%20feedback%20and%20more!-blue?style=for-the-badge&logo=discord)](https://discord.gg/S4tFu6Enne)
```

Same Discord link as https://github.com/sandboxie-plus/Sandboxie/commit/41e1ce99febaff2d4a722bcbf79346e85d232214

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
...
+
 [![Markdown links Status](https://github.com/sandboxie-plus/sandboxie-docs/actions/workflows/action.yml/badge.svg)](https://github.com/sandboxie-plus/sandboxie-docs/actions/workflows/action.yml) [![Codespell Status](https://github.com/sandboxie-plus/sandboxie-docs/actions/workflows/codespell.yml/badge.svg)](https://github.com/sandboxie-plus/sandboxie-docs/actions/workflows/codespell.yml)
```
Comment (isaak654) [inline README.md]:
```suggestion

[![Markdown links Status](https://github.com/sandboxie-plus/sandboxie-docs/actions/workflows/action.yml/badge.svg)](https://github.com/sandboxie-plus/sandboxie-docs/actions/workflows/action.yml) [![Codespell Status](https://github.com/sandboxie-plus/sandboxie-docs/actions/workflows/codespell.yml/badge.svg)](https://github.com/sandboxie-plus/sandboxie-docs/actions/workflows/codespell.yml)

[![Join our Discord Server](https://img.shields.io/badge/Join-Our%20Discord%20Server%20for%20bugs%20,%20feedback%20and%20more!-blue?style=for-the-badge&logo=discord)](https://discord.gg/S4tFu6Enne)

```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Review by isaak654 — APPROVED** [review]

**Pull request merged**